### PR TITLE
Release 2.5-M1

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

File: news/_posts/2017-01-26-akka-2.5-M1-released.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+* akka-testkit and akka-multi-node-testkit
+* experimental modules, such as akka-typed and akka-contrib
+* features, classes, methods that were deprecated in 2.4.0 or earlier and removed in 2.5.x
+
+Being binary compatible means that applications and libraries built on top of Akka 2.4.x continue to work with Akka 2.5.x without recompilation (subject to the conditions below), which implies that Akka HTTP as well as Play Framework 2.5 can be combined with Akka 2.5.
+
+### Migration Guide
+
+When migrating a code base to Akka 2.5 please refer to the[ migration guide](http://doc.akka.io/docs/akka/2.5-M1/project/migration-guide-2.4.x-2.5.x.html) in order to profit from some of the improvements.
```
Comment (2m) [inline news/_posts/2017-01-26-akka-2.5-M1-released.md]: Typo in the migration guide link text.

**Comment by ktoso** [comment]
LGTM, I hope the stats are correct :)

File: news/_posts/2017-01-26-akka-2.5-M1-released.md (hunk 1 above)
Comment (jrudolph) [inline news/_posts/2017-01-26-akka-2.5-M1-released.md]: MInor: blank at the wrong position in `the[ migration]`

Comment (patriknw) [inline news/_posts/2017-01-26-akka-2.5-M1-released.md]: ah, the link is correct but I see the[ mi

**Comment by jrudolph** [comment]
LGTM

File: news/_posts/2017-01-26-akka-2.5-M1-released.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+`AbstractActor` has been promoted from its experimental state and while doing this we did some small, but important, improvements to the API that will require some mechanical changes of your source code. See the [Actor (Java)](http://doc.akka.io/docs/akka/2.5-M1/project/migration-guide-2.4.x-2.5.x.html#Actor__Java_) section in the migration guide.
+
+It is possible to do a rolling update from Akka 2.4.16 to 2.5-M1 but some configuration changes might be needed first. See the [Rolling Update](http://doc.akka.io/docs/akka/2.5-M1/project/migration-guide-2.4.x-2.5.x.html#Rolling_Update)  section in the migration guide.
+
+Akka Persistence plugins released for 2.4 might not be compatible with the changes that were done in the API for Persistence Query in 2.5. We kindly ask the Akka Persistence plugin maintainers to migrate and release for 2.5. See the [Persistence Query](http://doc.akka.io/docs/akka/2.5-M1/project/migration-guide-2.4.x-2.5.x.html#Persistence_Query) section in the migration guide.
+
+### Credits
+
+The complete list of closed tickets can be found on the[ 2.5-M1](https://github.com/akka/akka/milestone/32?closed=1) milestones on github.
```
Comment (2m) [inline news/_posts/2017-01-26-akka-2.5-M1-released.md]: Same whitespace in the link text here as well.

File: news/_posts/2017-01-26-akka-2.5-M1-released.md (hunk 1 above)
Comment (patriknw) [inline news/_posts/2017-01-26-akka-2.5-M1-released.md]: fixed at two places, strange it must have been an err by the markdown converter

**Pull request merged**